README.md

```diff
@@ -85,14 +85,15 @@ npm install --save-dev svelte-dnd-action
+| Name                      | Type           | Required?                                                    | Default Value                                     | Description                                                  |
+| ------------------------- | -------------- | ------------------------------------------------------------ | ------------------------------------------------- | ------------------------------------------------------------ |
+| `items`                   | Array<Object>  | Yes. Each object in the array **has to have** an `id` property with a unique value (within all dnd-zones of the same type) | N/A                                               | The data array that is used to produce the list with the draggable items (the same thing you run your #each block on) |
+| `flipDurationMs`          | Number         | No                                                           | `0`                                               | The same value you give the flip animation on the items (to make them animated as they "make space" for the dragged item). Set to zero or leave out if you don't want animations |
+| `type`                    | String         | No                                                           | Internal                                          | dnd-zones that share the same type can have elements from one dragged into another. By default, all dnd-zones have the same type |
+| `dragDisabled`            | Boolean        | No                                                           | `false`                                           | Setting it to true will make it impossible to drag elements out of the dnd-zone. You can change it at any time, and the zone will adjust on the fly |
+| `dropFromOthersDisabled`  | Boolean        | No                                                           | `false`                                           | Setting it to true will make it impossible to drop elements from other dnd-zones of the same type. Can be useful if you want to limit the max number of items for example. You can change it at any time, and the zone will adjust on the fly |
+| `dropTargetStyle`         | Object<String> | No                                                           | `{outline: 'rgba(255, 255, 102, 0.7) solid 2px'}` | An object of styles to apply to the dnd-zone when items can be dragged in to it. Note: the styles override any inline styles applied to the dnd-zone. When the styles are removed, any original inline styles will be lost |
+| `transformDraggedElement` | Function       | No                                                           | `undefined`                                       | A function that is invoked when the draggable element is enters the dnd-zone. It passes in the dragged element as the first parameter, and the data of item from the items array as the second parameter. This allows you to override any fields on the element, such as changing the innerHTML. |
```

Comment:
1. `is enters` is probably a typo.
2. it is invoked every time the dragged element enters the list or hovers over a new index in the target list

Reply:
Good catch, updated.